### DISCUSSION TITLE
Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.0.0:copy

### DISCUSSION THREAD

**Discussion by CvEngin** [eclipse-kura/kura#5325] [Q&A] [answered]
During the target compilation process, the following error occurred in the final step：
Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.0.0:copy (copy-bundles-for-publishing) on project p2-repo-tests-deps: Unable to find/resolve artifact.: Could not transfer artifact org.hamcrest:org.hamcrest:jar:1.1.0.v20090501071000 from/to Metrics (https://mvnrepository.com/artifact/): Access denied to: https://mvnrepository.com/artifact/org/hamcrest/org.hamcrest/1.1.0.v20090501071000/org.hamcrest-1.1.0.v20090501071000.jar , ReasonPhrase:Forbidden.

**Comment by mattdibi** [comment, marked as answer]
Duplicate of https://github.com/eclipse/kura/discussions/5322
